userguide/kaui/includes/tenants.adoc

```diff
@@ -1,54 +1,338 @@
+
+a| * <<View the raw XML catalog,View the Raw XML Catalog>>
+* <<Delete a catalog,Delete a Catalog>>
+* <<Download the Catalog,Download the catalog>>
+
+|===
+
+On this tab, you can also <<Add a Currency to a Plan,add a currency to a plan>>.
+
+[NOTE]
+*Note:* You cannot edit a raw XML catalog in Kaui.
+
+==== Additional Resources
+
+* https://killbill.github.io/slate/#catalog[Catalog API]
+* https://docs.killbill.io/latest/userguide_subscription.html#components-catalog["Catalog"] section in _Subscription Guide_
+
+==== View Catalog Products and Plans
+
+You can view the current catalog's products and plans on the Catalog Show tab located at the bottom of the Tenant Configuration page.
+
+To get there:
+
+. Click on your username and tenant name in the upper right corner:
++
+image::Users-SelectTenantName.png[width=650,align="center"]
++
+Kaui displays the Tenant Configuration page.
++
+. Scroll down until you see the Catalog Show tab:
++
+image::Catalog-Tabs-ShowCat.png[width=850,align="center"]
+
+==== Create a Simple Catalog
+
+. <<View Catalog Products and Plans,Go to the Catalog Show tab>> on the Tenant Configuration page.
++
+image::Catalog-Tabs-ShowCat.png[width=850,align="center"]
++
+. Click the plus sign ( image:i_PlusGreen.png[] ) next to *Existing Plans*.
++
+Kaui opens the Catalog Configuration screen:
++
+image::CatalogConfiguration.png[width=450,align="center"]
++
+. Fill in the fields. For field descriptions, see <<Catalog Configuration Field Descriptions>>.
+. Click the *Save* button.
+
+==== Catalog Configuration Field Descriptions
+
+For additional field information, see the https://killbill.github.io/slate/#catalog[Catalog API].
+
+[cols="1,3"]
+|===
+^|Field ^|Description
+
+| Product Category
+a| Define whether this product is of the `BASE`, `ADDON`, or `STANDALONE` category:
+
+* Base products can have one or more addons.
+
+* Add-on products can be bundled with a base product.
+
+* Standalone products cannot have any add-ons.
+
+| Product Name
+| The name assigned to the product you are selling.
+
+| Plan Name
+| The name of the plan, which defines how the product will be sold (for example, a monthly subscription or a one-time purchase).
+
+| Amount
+| The price of the plan to be paid every billing period.
+
+| Currency
+| The currency this plan uses. If you need to add more currencies, you can do so by <<Add a Currency to a Plan,adding a currency to a plan>>.
+
+| Billing Period
+| The period for which the customer is billed.
+
+| Trial Length
+| Along with a unit of time (*Trial Time Unit* below), defines the length of the trial.
+
+| Trial Time Unit
+| Along with the *Trial Length* number above, specifies the time interval for the trial.
+|===
+
+==== Add a Currency to a Plan
+
+_I'm having trouble with this functionality. I get the error message "Error while creating plan: plan [NAME] already exists." There might be some rules around creating a new currency. Maybe can't do it with a simple plan?_
+
```

Comment:
Yes, I see this issue as well, I tried a couple of options but none of them seem to work - I have created [this issue ](https://github.com/killbill/technical-support/issues/47)to track it.

Reply:
Thank you!